doc/manuals/devel/mongoBackend.md

```diff
@@ -587,94 +593,94 @@ typedef struct MongoConnection
 
 [Top](#top)
 
 #### `processConditionVector()`
 
-This function is called at context subscription creation/update and eventually sent an initial notification associated to such subscription.
+This function is called during context subscription creation/update and possibly sends an initial notification associated to the subscription.
 
 <a name='flow-md-03'></a>
 ![`processConditionVector()` function detail](images/Flow-MD-03.png)
 
 _MD-03: `processConditionVector()` function detail_
 
-* `processConditionVector()` is invoked (step 1). See diagrams [MB-11](#flow-mb-11) and [MB-12](#flow-mb-12).
-* A loop iterates on each individual condition in the `NotifyConditionVector` vector (although most of the times this vector has only one item):
-   * The `processOnChangeConditionForSubscription()` is called to process the individual condition (step 2).
-   * `entitiesQuery()` is called to get the entities to be included in the notification (step 3), which in sequence relies on `collectionRangedQuery()` in the `connectionOperations` module in order to get the entities from DB (steps 4 and 5).
-   * The `pruneContextElements()` function is call in order to remove not found elements, as it doesn't make sense to include them in the notification (step 6).
-   * If after pruning there is some entity to send, steps 7 to 11 are executed.
-	   * In the case of conditions for particular attributes (i.e. not empty condition), then a second lookup is done using `entitiesQuery()` (steps 7, 8 and 9, plus prunning in step 10).
-	   * Notification is sent (step 11) using the `Notifier` object (from [ngsiNotify](README.md#srclibngsinotify) library) in order to actually sent the notification (step 3). The detail is provided in diagrams [NF-01](README.md#flow-nf-01) or [NF-03](README.md#flow-nf-03). In the case of conditions for particular attributes, notification is sent only if the previous check was ok. In the case of all-attributes notifications (i.e. empty codition) notification is always sent.
+* `processConditionVector()` (step 1) is invoked by mongoBackend functions. See diagrams [MB-11](#flow-mb-11) and [MB-12](#flow-mb-12).
+* A loop iterates over each individual condition in the `NotifyConditionVector` vector (although most of the times this vector has only one item):
+   * `processOnChangeConditionForSubscription()` is called to process the individual condition (step 2).
+   * `entitiesQuery()` is called to get the entities to be included in the notification (step 3), which in sequence relies on `collectionRangedQuery()` in the `connectionOperations` module in order to get the entities from the database (steps 4 and 5).
+   * `pruneContextElements()` is called in order to remove not found elements, as it makes no sense including them in the notification (step 6).
+   * If, after pruning, there is any entity to send, steps 7 to 11 are executed.

```

Comment:
Looks good in https://github.com/telefonicaid/fiware-orion/blob/task/proof_read/doc/manuals/devel/mongoBackend.md